Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "koji"
-version = "3.0.0"
+version = "4.0.0"
```

Comment:
Force release minor version instead, the fields may be public but since it's a binary that's irrelevant